.gitignore

```diff
@@ -17,3 +17,4 @@ Homestead.yaml
 npm-debug.log
 yarn-error.log
 coverage.xml
+.idea
```

Comment:
Ignores ide files of jetbrains IDEs

Reply:
Please use a global gitignore to ignore your IDE files instead: https://stackoverflow.com/questions/7335420/global-git-ignore